Michelle.deng/update logs archives encryption info (#27497)
DataDog/documentation: content/en/logs/log_configuration/archives.md

{{% /tab %}}
{{< /tabs >}}

#### Server-side encryption (SSE) for S3 archives

When creating or updating an S3 archive in Datadog, you can optionally configure **Advanced Encryption**. Three options are available under the **Encryption Type** dropdown:

- **Default S3 Bucket-Level Encryption** (Default): Datadog does not override your S3 bucket's default encryption settings.
- **Amazon S3 managed keys**: Forces server-side encryption using Amazon S3 managed keys ([SSE-S3][1]), regardless of the S3 bucket's default encryption.

> - **AWS Key Management Service**: Forces server-side encryption using a customer-managed key (CMK) from [AWS KMS][2], regardless of the S3 bucket's default encryption. You will need to provide the CMK ARN. 

Reviewer: ```suggestion
- **AWS Key Management Service**: Forces server-side encryption using a customer-managed key (CMK) from [AWS KMS][2], regardless of the S3 bucket's default encryption. You will need to provide the CMK ARN.
```

[1]: https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingServerSideEncryption.html
[2]: https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingKMSEncryption.html
{{< tabs >}}
{{% tab "Default S3 Bucket-Level Encryption" %}}

When this option is selected, Datadog does not specify any encryption headers in the upload request. The default encryption from your S3 bucket will apply.

To set or check your S3 bucket's encryption configuration:

1. Navigate to your S3 bucket.
2. Click the **Properties** tab.
3. In the **Default Encryption** section, configure or confirm the encryption type. If your encryption uses [AWS KMS][1], ensure that you have a valid CMK and CMK policy attached to your CMK.

[1]: https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingKMSEncryption.html

{{% /tab %}}
{{% tab "Amazon S3 managed keys" %}}

> This option ensure that all archives objects are uploaded with [SSE_S3][1], using Amazon S3 managed keys. This overrides any default encryption setting on the S3 bucket. 

Reviewer: ```suggestion
This option ensures that all archives objects are uploaded with [SSE_S3][1], using Amazon S3 managed keys. This overrides any default encryption setting on the S3 bucket. 
```

[1]: https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingServerSideEncryption.html
{{% /tab %}}
{{% tab "AWS Key Management Service" %}}

> This option ensure that all archives objects are uploaded using a customer-managed key (CMK) from [AWS KMS][1]. This overrides any default encryption setting on the S3 bucket. 

Reviewer: ```suggestion
This option ensures that all archives objects are uploaded using a customer-managed key (CMK) from [AWS KMS][1]. This overrides any default encryption setting on the S3 bucket. 
```

3. Go to the **Properties** tab in your S3 bucket and select **Default Encryption**. Choose the "AWS-KMS" option, select your CMK ARN, and save.

For any changes to existing KMS keys, reach out to [Datadog support][3] for further assistance.

> Users can also set encryption directly in the object upload request by configuring encryption settings. Optionally specify the encryption when you create or update an S3 archive via [Logs Archives API][4].

Reviewer 2: ```suggestion
You can also set encryption directly in the object upload request by configuring encryption settings. Optionally, specify the encryption when you create or update an S3 archive through the [Logs Archives API][4].
```

[1]: https://docs.aws.amazon.com/AmazonS3/latest/userguide/default-bucket-encryption.html
[2]: https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingKMSEncryption.html
[3]: /help/

> [4]: https://docs.datadoghq.com/api/latest/logs-archives/

Reviewer 2: ```suggestion
[4]: /api/latest/logs-archives/
```
